src/course-unit/clipboard/paste-notification/index.jsx

```diff
@@ -0,0 +1,110 @@
+import { useState } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Error as ErrorIcon,
+  Info as InfoIcon,
+  Warning as WarningIcon,
+} from '@edx/paragon/icons';
+import { useIntl } from '@edx/frontend-platform/i18n';
+
+import AlertMessage from '../../../generic/alert-message';
+import { ActionButton, AlertContent } from './components';
+import { getAlertStatus } from './utils';
+import { initialNotificationAlertsState } from './constants';
+import messages from './messages';
+
+const PastNotificationAlert = ({
+  staticFileNotices,
+  courseId,
+}) => {
```

Comment:
[optional] Can be simplified to one line
```suggestion
const PastNotificationAlert = ({ staticFileNotices, courseId }) => {
```